subprojects/gdk_rust/gdk_electrum/src/lib.rs

```diff
@@ -144,6 +152,12 @@ fn notify_fee(notif: NativeNotif, fees: &[FeeEstimate]) {
     let data = json!({"fees":fees,"event":"fees"});
     notify(notif, data);
 }
+fn notify_updated_txs(notif: NativeNotif) {
+    // This is used as a signal to trigger syncina via get_transactions, the transaction
```

Comment:
```suggestion
    // This is used as a signal to trigger syncing via get_transactions, the transaction
```

Reply:
Thanks, fixed in https://github.com/Blockstream/gdk/pull/59/commits/a903cb5d7f6723892621a5b668947ba8eaa1f6a9.